third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt

```diff
@@ -2,11 +2,11 @@ Tests that exporting works.
 
 ++++++++ testExportHtml
 
-# of .lh-audit divs (original): 136
+# of .lh-audit divs (original): 135
```

Comment:
@patrickhulce idk how this was missed from #11328 ...

Reply:
hm that's weird that PR passed and the last two master ones have passed https://github.com/GoogleChrome/lighthouse/runs/1033591612

I would've expected this change to fail...

does this get run with just `yarn build-devtools && yarn test-devtools` locally?

Reply:
yea (and `yarn update:test-devtools` to rebaseline)

Reply:
![image](https://user-images.githubusercontent.com/4071474/91369817-54330680-e7d2-11ea-8e69-72a8358b637f.png)
 hm.... (the one you linked)

Reply:
`run_web_tests.py` didnt exit 1 ...

Reply:
![image](https://user-images.githubusercontent.com/4071474/91380892-b0565480-e7eb-11ea-8b0d-4f8d524d3c13.png)

https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/test/webtests/TestExpectations

I'll look into re-enabling it.